pkg/kdump/kdump.scss

```diff
@@ -28,3 +28,7 @@
   max-block-size: 200px;
   overflow-y: auto;
 }
+
+.ct-monospace {
+  font-family: var(--pf-v5-global--FontFamily--monospace);
+}
```

Comment:
Why is this here? `pf-v5-u-font-family-monospace` and `pf-v5-u-font-family-monospace-vf` exist. See https://www.patternfly.org/utility-classes/text

Reply:
Fixed:
![Screenshot from 2023-08-17 14-29-47](https://github.com/cockpit-project/cockpit/assets/42733240/c8e453ee-af3b-4ed6-91bd-abff084099f3)
![Screenshot from 2023-08-17 14-30-10](https://github.com/cockpit-project/cockpit/assets/42733240/53b9d56d-dd1c-4f99-9215-81ca254d3030)
![Screenshot from 2023-08-17 14-29-59](https://github.com/cockpit-project/cockpit/assets/42733240/958ff9e9-4e95-4414-852f-94edc1b898f4)